genesis/engine/scene.py

```diff
@@ -566,14 +567,16 @@ def add_camera(
             Whether to denoise the camera's rendered image. Only available when using the RayTracer renderer. Defaults
             to True. If OptiX denoiser is not available in your platform, consider enabling the OIDN denoiser option
             when building the RayTracer.
+        debug : bool
+            Whether to use the debug camera (Rasterizer). Effective for BatchRenderer and RayTracer settings. Defaults to False.
```

Comment:
Can you clarify the documentation? At this point I have no clue what it is supposed to do.

You should explain that it enables to create cameras that can used to monitor / debug the simulation without being part of the that are not officially considered as part of the "sensors". Their output is rendered by the usual simple `Rasterizer` systematically, no matter if BatchRender and RayTracer is enabled. This way, it is possible to record the simulation with arbitrary resolution and camera pose, without interfering with what robots can perceive from their environment.